src/groupByKey.ts

```diff
@@ -2,6 +2,20 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<I>(
+  items: I[],
+  key: keyof I,
+): GroupsMap<I> {
+  const groupList: GroupsMap<I> = {};
+
+  items.forEach((el) => {
+    const listKey: string = `${el[key]}`;
```

Comment:
примитивы не указываются
```suggestion
    const listKey = `${el[key]}`;
```